pymtl3/passes/sverilog/translation/TranslationPass.py

```diff
@@ -0,0 +1,43 @@
+#=========================================================================
+# TranslationPass.py
+#=========================================================================
+# Author : Peitian Pan
+# Date   : March 12, 2019
+"""Translate a PyMTL component hierarhcy into SystemVerilog source code."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+from pymtl3.passes.BasePass import BasePass, PassMetadata
+
+from .SVTranslator import SVTranslator
+
+
+def mk_TranslationPass( _SVTranslator ):
```

Comment:
Instead of directly using SVTranslator in the pass, why do you need this function to make a pass?

Reply:
For testing purposes just like what we did for translators (I'll add the test cases after the import pass PR). I should have added more comments here...